community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/fusion/FusionSchemaIndexProvider.java

```diff
@@ -126,6 +130,27 @@ public InternalIndexState getInitialState( long indexId, IndexDescriptor descrip
     }
 
     @Override
+    public IndexCapability getCapability( IndexDescriptor indexDescriptor )
+    {
+        IndexCapability nativeCapability = nativeProvider.getCapability( indexDescriptor );
+        IndexCapability luceneCapability = luceneProvider.getCapability( indexDescriptor );
+        return new UnionIndexCapability( nativeCapability, luceneCapability )
+        {
+            @Override
+            public IndexOrder[] orderCapability( ValueGroup... valueGroups )
+            {
+                // No order capability when combining results from different indexes
+                if ( valueGroups.length == 1 && valueGroups[0] == ValueGroup.UNKNOWN )
```

Comment:
We need fusion to make this decision about ordering because he's responsible for combining the results and thus affect the ordering. The value capability can be left completely to underlying implementations and are combined with `UnionIndexCapability`.